### ISSUE TITLE
Python high cpu when enabling Subtitle Module

### ISSUE THREAD

**Issue by echel0n** [SiCKRAGETV/sickrage-issues#44]
<a href="https://github.com/bitzorro"><img src="https://avatars.githubusercontent.com/u/10099360?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [bitzorro](https://github.com/bitzorro)**
_Saturday Dec 06, 2014 at 18:09 GMT_
_Originally opened as https://github.com/SiCKRAGETV/SickRage/issues/1031 (5 comment(s))_

----

[ISSUE] Python high Cpu.
Windows 7
BRANCH: (develop) / COMMIT: (0365cc201b81c9df639d146f99ce2bd7dd972aa7) 
2.7.8 (default, Jun 30 2014, 16:03:49) [MSC v.1500 32 bit (Intel)]

Python runs with very high cpu , constant 50 %.
Log file can be found here: http://pastebin.com/cBDE3TN5

**Comment by bitzorro** [comment]
Using the latest develop branch from 24/12-14 and running with subtitles enabled and the cpu usage is Normal.

**Comment by bitzorro** [comment]
Using the latest develop branch from 24/12-14 and running with subtitles enabled and the cpu usage is Normal.

**Comment by bitzorro** [comment]
Earlyer i was uing 4 subtitle-Plugins, now i only use Opensubtitle-plugin.

**Comment by fernandog** [comment]
@bitzorro what's the the cpu usage now?

**Comment by bitzorro** [comment]
@fernandog. 0 to max 5 %

**Comment by bitzorro** [comment]
Small Peaks at 15 % when sickrage starts.

**Comment by fernandog** [comment]
@bitzorro maybe SR should limit to 1 or 2 enabled subtitle plugins.

**Comment by markheloking** [comment]
Closing this as this doesn't seem to be an issue anymore.

**Comment by fernandog** [comment]
@markheloking the issue still exists. re-open

https://github.com/SiCKRAGETV/sickrage-issues/issues/370

**Comment by fernandog** [comment]
@SunMar maybe with new version, should fix this?

**Comment by SunMar** [comment]
not sure, haven't noticed the issue yet myself, but I'll keep an eye out for it.

**Comment by perler** [comment]
another thought: I see, that sickrage re-downloads some existing subtitles.
at least I get a notfication about it (but the subtitle is here since
november). could this habe something to do with the CPU issue?

**Comment by fernandog** [comment]
Maybe it scan always all the folders

**Comment by perler** [comment]
but shouldn't we see this in the logs?

**Comment by fernandog** [comment]
not sure

**Comment by perler** [comment]
ok, I did some research:

the problem looks like a problem with filenames. the high CPU usage
happended in this directory:

https://gist.github.com/anonymous/f7cbf1302ab92100afa2

beforehand I removed a Season 8 of Midsomer Murders which had the same
problem, but after removing Season 16 I left Season 17 which causes no
problem. The subtitles backlog is now worded on and I wait for another
hitch.

what separates Season 8/16 from 17 is, that the former was not downloaded
by sickrage but transferred from other sources, even a different filesystem
so it could be some character conversion problem. but on the other hand, a
lot of other directories from the same source don't make any trouble and
download subtitles quite fine.
